blocks/page_list/templates/itinerary/ItineraryStore.js

```diff
@@ -1,59 +1,122 @@
 import { dispatch, register } from './ItineraryDispatcher';
 import { EventEmitter } from 'events';
 import ItineraryConstants from './ItineraryConstants';
```

Comment:
we may have naming collisions so:

import {Actions:ItineraryActions} from './ItineraryConstants';
import {Actions:WalkActions} from './WalkConstants'